pages/public_cloud/public_cloud_network_services/getting-started-01-create-lb-service/guide.en-gb.md

```diff
@@ -1,48 +1,79 @@
+- (Optional): This guide explains the load balancer configuration through Graphical Interface & Command Line Interface. If you want to use the latter, then install the [OpenStack Command Line environment](/pages/public_cloud/compute/prepare_the_environment_for_using_the_openstack_api)
 
 ## Instructions
 
-### Configuring your private network
+### Creating the Load Balancer from the OVHcloud Control Panel
 
-Before creating a Load Balancer, you will need to set up a private network:
+Open your Public Cloud project and click `Load Balancer`{.action} (under `Network`) in the left menu, then click the `Create a Load Balancer`{.action} button.
 
-```bash
-openstack network create my_network
+The configuration page will open.
 
-openstack subnet create my_subnet --subnet-range <my_private_ip_range/mask> --network my_network --no-dhcp
+#### Step 1 : Size choice
 
-openstack router create my_router
+![Size choice](images/size.png){.thumbnail}
 
-openstack router add subnet my_router my_subnet
+The interface contains a link to the website on which the characteristics / benchmark of all size are provided. Once you have chosen your size, click `Next`{.action}.
 
-openstack router set --external-gateway Ext-Net my_router
-```
+#### Step 2 : Region choice
 
-You can now attach your instances to the new network. We recommend following our guide to [integrate an instance into vRack](/pages/public_cloud/public_cloud_network_services/getting-started-07-creating-vrack#instance-integration). List the addresses of your instances in your network with the following command:
+![Region choice](images/region.png){.thumbnail}
 
-```bash
-openstack server list
-```
+Only regions on which you have a private network and at least one subnet can be selected. Select the region and click `Next`{.action}.
+
+#### Step 3 : Attach a public IP (or not)
+
+![Public IP choice](images/floating_IP.png){.thumbnail}
+
+At this stage, you need to know if your Load Balancer will be receiving public traffic or not (for more details, read our [Public Cloud Networking concepts](/pages/public_cloud/public_cloud_network_services/concepts-01-public-cloud-networking-concepts) page). 
+
+If your Load Balancer is processing public traffic, you have 2 options :
+
+- `New Public IP`: this option will create a new Floating IP for your Load Balancer. 
+- If you already have a Floating IP, the interface will offer you to pick one.
+
+If your Load Balancer is processing private traffic, choose `No Public IP`.
+
+#### Step 4 : Select the private network and the subnet where the Load Balancer will be spawned
+
+![Network choice](images/private_network.png){.thumbnail}
+
+The interface will inform you if the private network / subnet is not compliant with the prerequisites (see [Public Cloud Networking concepts](concepts-01-public-cloud-networking-concepts/guide.en-gb.md###Network pre requisites)).
```

Comment:
This link won't work as:
- format is not compliant (see the other links for reference)
- the target (Network pre requisites) does not exist